shared/styles/index.desktop.js

```diff
@@ -106,14 +106,9 @@ export const backgroundURL = (...to: Array<string>) => {
 
 export const hairlineWidth = 1
 export const styleSheetCreate = (obj: Object) => obj
-export const collapseStyles = (styles: Array<Object> | Object) => {
-  if (isArray(styles)) {
-    return styles.reduce((map, item) => {
-      return {...map, ...item}
-    }, {})
-  } else {
-    return styles
-  }
+export const collapseStyles = (styles: $ReadOnlyArray<CollapsibleStyle>): Object => {
```

Comment:
why is this in 3 different prs?  i don't understand how all these reconcile against each other

Reply:
Sorry, I had rebased this PR and https://github.com/keybase/client/pull/11151 on top of the `collapseStyles` PR to make sure that it type-checked -- not sure why GitHub is still showing this as a diff, but now that I merged the `collapseStyles` PR in, it should disappear.